tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -195,10 +205,10 @@ def _init_devices(self):
                 port, _ = device
                 try:
                     with OpenThreadController(port) as otc:
-                        logger.info('Resetting %s', port)
+                        logger.info("Resetting %s", port)
```

Comment:
If you want to keep single quote,  The  option`-S` could be appended when run `black` tool

> -S, --skip-string-normalization
>                                   Don't normalize string quotes or prefixes.
>